controllers/analytics.go

```diff
@@ -55,6 +58,44 @@ func getNewUUID(p *api.Pattern) string {
 	return newuuid
 }
 
+func getSimpleDomain(p *api.Pattern) string {
+	parts := strings.Split(p.Status.ClusterDomain, ".")
+	simpleDomain := strings.Join(parts[len(parts)-2:], ".")
```

Comment:
Is 2 too few?  3?

Reply:
... now that we can compute a subset

Reply:
I think 2 is fine? We also have the device hash now to distinguish the inner bits